examples/00_published/plot_booky_1D_time_freq_inv.py

```diff
@@ -264,7 +264,7 @@ def run(plotIt=True, saveFig=False, cleanup=True):
     # Bird height from the surface
     b_height_skytem = skytem["src_elevation"].value
     src_height = b_height_skytem[rxind_skytem]
-    srcLoc = np.array([[0., 0., src_height]])
+    srcLoc = np.array([0., 0., src_height])
```

Comment:
This is changed from previous version. Now we need to put 1d array for src location. 
This seems consistent with frequency domain code. Is that what you aimed for?

Reply:
hmm, k I will take a look. Ideally, we should be able to input either.

Reply:
yeah, now we require it to be a vector. If you think this is annoying, I can add the functionality to allow a 2D array to be input and it be cast to a vector. Let me know what you think @sgkang

Reply:
Will be nice, if we can input both forms.

Reply:
kk, I have added a property that will allow us to do that: `LocationVector` in `Props.py`. This is related to: seequent/properties#250